src/Paramore.Brighter.Extensions.DependencyInjection/ServiceCollectionBrighterBuilder.cs

```diff
@@ -103,8 +103,8 @@ public IBrighterBuilder AsyncHandlersFromAssemblies(params Assembly[] assemblies
         public IBrighterBuilder AutoFromAssemblies(params Assembly[] extraAssemblies)
         {
             var appDomainAssemblies = AppDomain.CurrentDomain.GetAssemblies().Where(a =>
-                !a.IsDynamic && !a.FullName.StartsWith("Microsoft.", true, CultureInfo.InvariantCulture) &&
-                !a.FullName.StartsWith("System.", true, CultureInfo.InvariantCulture));
+                !a.IsDynamic && a.FullName?.StartsWith("Microsoft.", true, CultureInfo.InvariantCulture) != true &&
+                a.FullName?.StartsWith("System.", true, CultureInfo.InvariantCulture) != true);
```

Comment:
These changes are to satisfy nullability warnings. It seems mostly reasonable that if an assembly is not dynamic then it will have a `FullName`, but there's no harm in removing any chance of dereferencing a null here.